tests/release/services/release_plan_owner_ref.go

```diff
@@ -11,7 +11,7 @@ import (
 	releaseApi "github.com/redhat-appstudio/release-service/api/v1alpha1"
 )
 
-var _ = framework.ReleaseSuiteDescribe("[HACBS-2469]test-releaseplan-owner-ref-added", Label("release", "releaseplan-ownerref", "HACBS"), func() {
+var _ = framework.ReleaseServicesSuiteDescribe("[HACBS-2469]test-releaseplan-owner-ref-added", Label("release-services", "releaseplan-ownerref", "HACBS"), func() {
```

Comment:
Please use "ReleaseServiceSuiteDescribe" instead of "ReleaseServicesSuiteDescribe" and "release-service" instead of "release-services".